src/patternfly/components/ToggleGroup/toggle-group.scss

```diff
@@ -79,3 +85,11 @@
     pointer-events: none;
   }
 }
+
+.pf-c-toggle-group__icon + .pf-c-toggle-group__text {
+  margin-left: var(--pf-c-toggle-group__icon--text--MarginLeft);
+}
+
```

Comment:
This looks good, though I wonder if we would ever want the space to be different depending on which comes first. It seems like we wouldn't, so you could also combine the selectors and apply a single `margin-left` declaration. wdyt?

Reply:
@mcoker yeah agreed i think we would want the spacing to be the same. what do you mean by "combine the selectors and apply a single margin-left declaration"?